### ISSUE TITLE
document version 2.7.0 option 'IsZoomControlEnabled' demo description not match the code

### ISSUE THREAD

**Issue by lanser2077** [wailsapp/wails#3191]
### Have you read the Documentation Contribution Guidelines?

- [X] I have read the [Documentation Contribution Guidelines](https://wails.io/community-guide#ways-of-contributing).

### Description

The option 'IsZoomControlEnabled' is not on options.App, i find it in windows.Options struct.
document version 2.7.0 
https://wails.io/zh-Hans/docs/reference/options#应用程序参数选项 

### Self-service

- [x] I'd be willing to address this documentation request myself.

**Comment by leaanthony** [comment]
Yeah it was updated via a PR recently for English. It takes a while for translations to come through.
